src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown

```diff
@@ -283,7 +302,7 @@ Email content to: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMar
 
 * App Center Alliances: [concur_AppCenterAlliance@sap.com](mailto:concur_AppCenterAlliance@sap.com)
 * App Center Marketing: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMarketing@sap.com)
-* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com) and [arian.white@sap.com (ENT)](mailto:arian.white@sap.com)
+* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com)
 * SAP Concur Event Sponsorships: [concursponsorships@sap.com](mailto:concursponsorships@sap.com)
 * Client Activation: [partner_client_activation@concur.com](mailto:partner_client_activation@concur.com)
```

Comment:
This alias should change to: concur_partnerclientactivation@sap.com ​

Reply:
RoAnn -- Laurel has asked me to do some additional work to this page -- I will fix this there.